portal/eproms/templates/eproms/base.html

```diff
@@ -98,6 +98,11 @@ <h2 class="modal-title">{{ _("TrueNTH") }}</h2>
 <script src="https://unpkg.com/i18next-browser-languagedetector@2.0.0/i18nextBrowserLanguageDetector.min.js"></script>
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.6.1/js/bootstrap-datepicker.min.js" async></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.11.1/bootstrap-table.min.js"></script>
+<script src="{{ url_for('static', filename='js/validator.js') }}"></script>
```

Comment:
Can you use this [CDNJS-hosted version](https://cdnjs.com/libraries/1000hz-bootstrap-validator)?